deployment/dev/servers.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
Can we define these things also in the compose files via env variables?

If not, better name `pg-admin.config.json` or so.

Reply:
Hm I am not aware of a way to define this via env variables. In https://www.pgadmin.org/docs/pgadmin4/development/import_export_servers.html I only saw the option to use a JSON file.

I will call this file `pg-admin-servers.json`, because the term "servers" is used in the PGAdmin documentation (as linked above), and also internally (we copy this file to `pgadmin4/servers.json`).